README.md

```diff
@@ -102,6 +102,7 @@ a double dash to prevent input files being used as option arguments:
                                 sequences.
     --config-file <file>        Read `minify()` options from JSON file.
     -d, --define <expr>[=value] Global definitions.
+    --ecma <version>            Specifiy supported level of ECMAScript.
```

Comment:
I'll admit that the current default is hard to put into words & it feels inconsistent, but yet it follows Principle of Least Surprise.